### DISCUSSION TITLE
HelloGitHub 徽章计划

### DISCUSSION THREAD

**Discussion by 521xueweihan** [baidu/dperf#457] [General]
🎉 恭喜！您的[项目](https://hellogithub.com/repository/67958cc5d1f44a6a84f3544e3c007e5f)已被 HelloGitHub 社区收录和推荐，现邀请您加入 HelloGitHub 徽章计划。加入后，您将获得以下特权：

- 社区认可: 徽章代表你的项目已通过 HelloGitHub 社区的严格筛选，并获得推荐。
- 提高曝光: 佩戴徽章后，你的项目将获得更多推荐流量，吸引更多用户和贡献者。
- 增强互动: 用户可通过徽章快速了解项目，并与之互动（点赞、评论、收藏）。
- 收集反馈: 获取广大用户的真实反馈，持续优化你的项目。
- 特殊标识: 认证后，你的评论将有显著标识，并获得置顶优先权。

📌 [立即点击](https://hellogithub.com/repository/67958cc5d1f44a6a84f3544e3c007e5f/embed) 完成徽章佩戴，加入 HelloGitHub 徽章计划，让你的开源项目绽放更耀眼的光彩。

---

**HelloGitHub**，一个专注于发现、分享、推广开源项目的社区。自 2016 年创立以来，从每月一更的月刊发展到今天，已经成为一个拥有超过 **1 万** 用户的充满活力的开源社区。我们的足迹遍布多个内容平台，并获得了全网 **50 万** 粉丝的信赖与支持。

<img width="1245" alt="截屏2024-06-12 09 29 42" src="https://github.com/No-Chicken/OV-Watch/assets/8255800/7ac6b71e-f052-4f44-88b1-5c772e917348">

**Comment by pengjianzhang** [comment]
非常感谢hellogithub的邀请。
在readme上加入如下代码后，图片显示异常。是否可以提供没有图片的纯文字的链接，谢谢。

<a href="https://hellogithub.com/repository/67958cc5d1f44a6a84f3544e3c007e5f" target="_blank"><img src="https://api.hellogithub.com/v1/widgets/recommend.svg?rid=67958cc5d1f44a6a84f3544e3c007e5f&claim_uid=Thc9mJByaKSbdng" alt="Featured｜HelloGitHub" style="width: 250px; height: 54px;" width="250" height="54" /></a>

**Reply by 521xueweihan** [reply]
@pengjianzhang 您好网站的国际化和迷你徽章已上线，欢迎您再次认领。

<img width="1132" alt="image" src="https://github.com/user-attachments/assets/a7ea12c1-acb8-4ba0-9fb5-0aa60663aa05">

**Reply by pengjianzhang** [reply]
已经加到readme了，极简徽章非常好用

**Reply by 521xueweihan** [reply]
@pengjianzhang 感谢您的支持和认领～
